opentreemap/importer/errors.py

```diff
@@ -3,13 +3,18 @@
 from __future__ import unicode_literals
 from __future__ import division
 
-# TODO: stop storing error text in the database in order to
-# make lazy translation possible.
-# See https://github.com/OpenTreeMap/OTM2/issues/1880
-# from django.utils.translation import ugettext_lazy as trans
-trans = lambda x: x
+from django.utils.translation import ugettext_lazy as trans
+
+_messages_by_code = {}
+
+def e(code, message, fatal):
```

Comment:
I did try that, and thought it hurt readability. So I'm leaving it even though `_e` is more correct.